src/app/Reducers/userReducer.js

```diff
@@ -0,0 +1,22 @@
+import { ADD_FRIEND, REMOVE_FRIEND } from '../Constants/ActionTypes';
+
+const initialState = {
+  friends: [{id: 1989}]
+};
+
+export default function userReducer(state=initialState, action) {
+
+  switch (action.type) {
```

Comment:
handles the actions and have a default to always return a new state or the current one